cms/djangoapps/contentstore/views/videos.py

```diff
@@ -150,19 +151,76 @@ def videos_handler(request, course_key_string, edx_video_id=None):
         return videos_post(course, request)
 
 
+def validate_video_image(image_file):
+    """
+    Validates video image file.
+
+    Arguments:
+        image_file: The selected image file.
+
+   Returns:
+        error (String or None): If there is error returns error message otherwise None.
+    """
+    error = None
+    image_file_width, image_file_height = get_image_dimensions(image_file)
+    image_file_aspect_ratio = abs(image_file_width / float(image_file_height) - settings.VIDEO_IMAGE_ASPECT_RATIO)
+
+    if not all(hasattr(image_file, attr) for attr in ['name', 'content_type', 'size']):
```

Comment:
Yeah sorry https://studio-videoimage.sandbox.edx.org/videos/course-v1:edx+VideoTest+2017_R1.

These are backend messages but will be shown to user too.